lib/java/com/google/android/material/drawable/DrawableUtils.java

```diff
@@ -207,44 +230,52 @@ public static Drawable compositeTwoLayeredDrawable(
     }
     LayerDrawable drawable =
         new LayerDrawable(new Drawable[] {bottomLayerDrawable, topLayerDrawable});
+
+    final int topLayerOriginalWidth = topLayerDrawable.getIntrinsicWidth();
```

Comment:
nit)  name those variables as:

topLayerIntrinsicWidth, etc.

Reply:
I'll check why the tests are broken. Can you adjust the PR according to the comment first?

Reply:
> nit) name those variables as...

Done